backend/isoko-backend/src/handlers/business/put-edit-business-page.js

```diff
@@ -9,10 +14,50 @@ exports.putEditBusinessPageHandler = async (event) => {
    }
 
    console.info('received:', event);
+   const { businessId } = event.pathParameters; 
+
+   if (businessId == null) {
+      throw new Error(
+         `Missing query parameter 'businessId'. Request URL format: PUT/business/{businessId}`
+      );
+   }
+
+   const requestBody = event.body && JSON.parse(event.body);
+   const fieldName = _.get(requestBody, 'fieldName');
+   const newVal = _.get(requestBody, 'newVal');
```

Comment:
I think you might've misinterpreted the API spec for this specific endpoint. For this endpoint say I wanted to update the name and ownerName field in the business I would send a request as follows:
```
{
   name: "New name",
   ownerName: "New Owner Name",
}

```

Rather than explicitly stating the fieldName and newVal as you have it set up now. Let me know if that doesn't make sense and I can explain it better.

Reply:
ohhhh shoot yeah okay that totally makes sense, thanks! i think i just got confused by the [] notation in the spec and didn't realize the second line was an example